README.md

```diff
@@ -168,20 +154,18 @@ Se asume que la persona tiene conocimientos previos en:
     Copie y pegue:
     ```java
     @Test
-	public void testGet() {
-		
-		baseURI =  "https://reqres.in/api";
+	  public void testGet() {
+		RestAssured.baseURI = "https://reqres.in/api";
```

Comment:
From the beginning was bad (not your change), please move to be indented the method signature:
```java
@Test 
public void testGet() {
...
}
```